modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/asset/full_content.jsp

```diff
@@ -40,7 +40,12 @@ String entryTitle = BlogsEntryUtil.getDisplayTitle(resourceBundle, entry);
 				%>
 
 				<c:if test="<%= Validator.isNotNull(coverImageURL) %>">
-					<div aria-label="<%= HtmlUtil.escapeAttribute(HtmlUtil.stripHtml(entry.getCoverImageCaption())) %>" class="aspect-ratio aspect-ratio-8-to-3 aspect-ratio-bg-cover cover-image" role="img" style="background-image: url(<%= coverImageURL %>);"></div>
+
+					<%
+					String coverImageCaption = HtmlUtil.escapeAttribute(HtmlUtil.stripHtml(entry.getCoverImageCaption()));
+					%>
+
+					<div <c:if test="<%= Validator.isNotNull(coverImageCaption) %>"> aria-label="<%= coverImageCaption %>" role="img" </c:if> class="aspect-ratio aspect-ratio-8-to-3 aspect-ratio-bg-cover cover-image" style="background-image: url(<%= coverImageURL %>);"></div>
```

Comment:
I think that I see on jsps more the ternary operator 
`<div <%= Validator.isNotNull(coverImageCaption) ? "aria-label=\"" + coverImageCaption + "role=\"img\"" : StringPool.BLANK%> class="aspect-ratio aspect-ratio-8-to-3 aspect-ratio-bg-cover cover-image" style="background-image: url(<%= coverImageURL %>);"></div>`

than with this way 


![](https://media3.giphy.com/media/yj5oYHjoIwv28/giphy.gif)